src/utils/gfycatSdk.js

```diff
@@ -3,7 +3,11 @@
 
 import Gfycat from './gfycat-sdk.umd.bundle.min';
 
-export default new Gfycat({
-    client_id: '2_KtH_W5',
-    client_secret: '3wLVZPiswc3DnaiaFoLkDvB4X0IV6CpMkj4tf2inJRsBY6-FnkT08zGmppWFgeof',
-});
+let instance = null;
+
+export default function (key, secret) {
+    if (!instance) {
+        instance = new Gfycat({client_id: key, client_secret: secret});
```

Comment:
Does this cause any errors if you have this enabled but haven't configured the API keys?

Reply:
Any blank API key/secret now defaults to specific values so this case shouldn't happen.  Normally, the SDK throws a single, custom exception that doesn't affect operation if it isn't given credentials.